app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,46 @@
+function setToggle(elementClass, targetElement, classToAdd, lastIndex) {
+    const allDays = document.getElementsByClassName(elementClass);
+    for (let i = lastIndex; i < allDays.length; ++i) {
+        allDays[i].addEventListener("click", function () {
+            const target = document.getElementById(targetElement);
+            target.classList.toggle(classToAdd);
+        })
+    }
+}
+
+document.addEventListener(
+    'DOMContentLoaded', function () {
+        setToggle("day", "day-view", "day-view-visible", 0);
+    }
+)
+
+function loadWeek(lastDay, index) {
+    if (lastDay.dataset.last === "false") {
+        return false;
+    }
+    lastDay.dataset.last = false;
+    const path = '/calendar/month/' + lastDay.id;
+    alert(path);
```

Comment:
נו באמת אחי :S

Reply:
It was not me it was the "AYEFUT" ;) 
TOKAN.